dotCMS/src/main/java/com/dotcms/api/system/event/message/builder/SystemMessage.java

```diff
@@ -67,4 +69,23 @@ public String toString() {
                 ", type=" + type +
                 '}';
     }
+
+    @Override
+    public boolean equals(final Object another) {
+        if (this == another) return true;
+        if (another == null || getClass() != another.getClass()) return false;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This statement should have braces](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5642172)